### PR TITLE
Views

### PR DESCRIPTION
**Pull request by arabiu033**
## Overview

- Add the Views associated with the Models and Controllers I've already created for the Blog App.

## Workflow

- Match the giving wireframe view.

### CODE REVIEW

**Review by FrederickMih — COMMENTED** [review]
## Hi :raising_hand:  @arabiu033 , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that contains the following issues:
- The Future Features section of your README file is missing 
- Your app seems not to be working because of the migrations issues on the screenshot below
Screenshot from 2023-01-26 04-09-49


Your Code Review Requeft will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by arabiu033** [comment]
Hello, current reviewer :hand:

As you can see, I dropped the entire database and recreated it, as well as run the migrations, and I had no problems. Please let me know if you find any problems and how you think I can solve them because everything is fine here.

db

File: README.md [hunk 1]
```diff
@@ -192,14 +192,14 @@ Example:
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
```
Comment (t-yanick) [inline README.md:216]: - [ ] Good job with the ReadMe, but you have to make complete the steps to start your app locally. I don't think it is possible to start the server when the database has not been created and migrations run

**Review by t-yanick — CHANGES_REQUESTED** [review]
Hi @arabiu033 , 

### Good job so far!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages
:heavy_check_mark: All functionalities work just as per the wireframe
:heavy_check_mark:  Excellent job making use of partials to render all users rather than` @users.each`
:heavy_check_mark:  All linter checks passed.

There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (arabiu033) [reply README.md:216]: Thank you @t-yanick :heart: Update the readme file.

**Review by CollinsTatang — APPROVED** [review]
# Approved 👏  
### **Highlights 🚀 :heavy_check_mark: :heavy_check_mark:** 

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### **Congratulations!** 🎉 
image



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**